packages/@aws-cdk/aws-lambda/lib/function.ts

```
@@ -779,6 +786,19 @@ export class Function extends FunctionBase {
       throw new Error(`Ephemeral storage size must be between 512 and 10240 MB, received ${props.ephemeralStorageSize}.`);
     }
 
+    let rmc = undefined;
+    if (props.runtimeManagement) {
+      if (props.runtimeManagement.mode === UpdateRuntimeOn.MANUAL) {
+        rmc = {
+          runtimeVersionArn: props.runtimeManagement.arn,
```

we need to throw an error if `runtimeManagement.arn` is not defined